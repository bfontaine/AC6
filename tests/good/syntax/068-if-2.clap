val x = 
  if True then U

