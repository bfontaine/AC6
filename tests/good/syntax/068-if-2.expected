val x =
  (case { True => U | False => U { U } }) (True)
